README.md

```diff
@@ -1,5 +1,18 @@
 [![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
 
+Table of Contents
+=================
+  i. About
+  ---------
+  ii. Contributing
+  -----------------
+  iii. License
+  -----------------
+  iv. Installing
+  -----------------
+  v. More Information
+  -------------------
+
```

Comment:
If you're going to add a table of contents, it should link to the entries it lists.